module/zfs/vdev.c

```diff
@@ -52,15 +52,21 @@
 #include <sys/zvol.h>
 #include <sys/zfs_ratelimit.h>
 
-/* maximum number of metaslabs per top-level vdev */
+/* target number of metaslabs per top-level vdev */
 int vdev_max_ms_count = 200;
```

Comment:
I had the same thought, but in the end decided against it since we want to keep this consistent across implementations.  Plus it's unlikely users would ever need to tune this, it's primarily made available for developers.